src/prefect/orion/schemas/actions.py

```diff
@@ -63,13 +65,23 @@ class FlowCreate(ActionBaseModel):
     """Data used by the Orion API to create a flow."""
 
     name: str = FieldFrom(schemas.core.Flow)
+    description: Optional[str] = FieldFrom(schemas.core.Flow)
     tags: List[str] = FieldFrom(schemas.core.Flow)
 
+    @validator("description")
+    def validate_description_length(cls, description):
```

Comment:
👍 great. We'll need this on update too. I think that's why validators like `validate_block_type_slug` are declared at the top-level instead of within the class.

Reply:
Thank you! Some confusion about the validators is cleared up for me here:

I could only see each of those top level validators (validate_block_type_slug and validate_block_document_name)
being called once (in *Create actions), so I wasn't sure why they were defined at the top level. I see now that though they are only currently being used in one spot, pulling them out to the top level is the re-use pattern [suggested by pydantic](https://docs.pydantic.dev/usage/validators/#reuse-validators).

Thanks for the suggestion. I will pull out my validate_description_length validator to the top-level so that I can re-use it in the FlowUpdate action as well.